### DISCUSSION TITLE
Is it possible to use sandboxie as a compilation isolation environment?

### DISCUSSION THREAD

**Discussion by shutaozhenzhen** [sandboxie-plus/Sandboxie#4347] [General]
Is it possible to use sandboxie as a compilation isolation environment? I wanted to be able to completely erase the programs I installed during the process after compiling the software, and using a virtual machine for this seemed a little too expensive

**Comment by bastik-1001** [comment]
There was a [similar question in the forum](https://forum.xanasoft.com/threads/using-sandboxie-for-software-development.2580/), and it comes down to, whenever or not the tool chain can be started in a sandbox. As mentioned there, not everything runs smoothly.
